### ISSUE TITLE
how the demo.py use your model predict the result

### ISSUE THREAD

**Issue by ghost** [ZhengyuZhao/Adaptive-Discriminative-Region-Discovery#5]
demo.py use the model is places-CNN，and the secene result is :
`probs, idx = h_x.sort(0, True)`
but this model is pretrained model from placee365 (wideresnet18_places365.pth.tar)， where your model ?

**Comment by ZhengyuZhao** [comment]
> demo.py use the model is places-CNN，and the secene result is :
> `probs, idx = h_x.sort(0, True)`
> but this model is pretrained model from placee365 (wideresnet18_places365.pth.tar)， where your model ?

Hi, we only used the pre-trained model in this paper without finetuning it on the target dataset (SUN397). For the label information that is used for generating discriminative patches, please refer to the last two sentences in Section 3.1 of our paper! You can also follow other .py scripts for more details.

**Comment by ghost** [comment]
your code or paper's main task is to generate discriminative regions not scene recognition , is right ?

**Comment by ZhengyuZhao** [comment]
> your code or paper's main task is to generate discriminative regions not scene recognition , is right ?

Yes, the main contribution lies in an adaptive way to discover discriminative regions. And the insight is based on an intuitive conceptual understanding of scenes.

So the procedure is that we first generate the discriminative regions for each image, and then use th extracted features from these regions (including the global image) to train a SVM classifier for scene recognition.
